ui_tests/src/test/resources/com/saucedemo/account/signInPositive.feature

```diff
@@ -0,0 +1,23 @@
+@signIn
```

Comment:
@MAKELAQA This feature file is missing feature definition.
Every feature file as per the gherkin syntax must start with Feature: keyword:
See Feature section here: https://cucumber.io/docs/gherkin/reference/#feature

Reply:
Feature: Sign In
It was already there in '' FEATURE FILE ''
It should be on the top of each page meaning : SignInpositive.feature ,SignInNegative.feature etc....
Please let me know that I can go further steps !

Reply:
This: `Feature: Sign In` section of feature file was missing. You added it.
This one is resolved 👍